pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -443,15 +443,15 @@ func (k *Bootstrapper) needsReset(conf string, hostname string, port int, client
 	}
 
 	if err := kverify.ExpectAppsRunning(client, kverify.AppsRunningList); err != nil {
-		glog.Infof("needs reset: %v", err)
+		glog.Infof("needs reset because expected components to be running : %v", err)
 		return true
 	}
 
 	if err := kverify.APIServerVersionMatch(client, version); err != nil {
-		glog.Infof("needs reset: %v", err)
+		glog.Infof("needs reset because API server version match : %v", err)
```

Comment:
I think this can be reverted. The error made sense already. If not:

`needs reset: api version: %v`